lite/valor_lite/classification/computation.py

```diff
@@ -182,9 +182,9 @@ def compute_metrics(
         out=precision,
     )
 
-    accuracy = np.zeros_like(recall)
+    accuracy = np.zeros(n_scores, dtype=np.float64)
```

Comment:
> because we aren't considering labels
This is why haha

`recall` has shape `(n_scores, n_labels)` so we need to explicitly define size without a label count.